frame/vesting/src/benchmarking.rs

```diff
@@ -375,6 +375,49 @@ benchmarks! {
 		);
 	}
 
+	force_remove_vesting_schedule {
+		let l in 0 .. MaxLocksOf::<T>::get() - 1;
+		let s in 0 .. T::MAX_VESTING_SCHEDULES;
```

Comment:
Just an observation:  
We have multiple benchmarks with components that dont need dynamic inputs, like `T::WeightInfo::force_remove_vesting_schedule(MaxLocksOf::<T>::get(), T::MAX_VESTING_SCHEDULES)`.   
The only advantage that comes to my mind from this is that it makes it possible for devs to re-use the Substrate weights without benchmarking themselves...  
Otherwise it just slows down our benchmarking.